.github/workflows/configure_go.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
net 20 new lines are mostly comments in this file, but also this does more (ex fixing the go build cache), so hopefully ok